62_CAD/shaders/globals.hlsl

```diff
@@ -262,9 +294,15 @@ NBL_CONSTEXPR uint32_t InvalidRigidSegmentIndex = 0xffffffff;
 NBL_CONSTEXPR float InvalidStyleStretchValue = nbl::hlsl::numeric_limits<float>::infinity;
 
 
-// TODO[Przemek]: we will need something similar to LineStyles but related to heigh shading settings which is user customizable (like LineStyle stipple patterns) and requires upper_bound to figure out the color based on height value.
+// TODO[Przemek]: we will need something similar to LineStyles but related to heigh shading settings which is user customizable (like  stipple patterns) and requires upper_bound to figure out the color based on height value.
 // We'll discuss that later or what it will be looking like and how it's gonna get passed to our shaders.
 
+struct TriangleMeshVertex
+{
+    pfloat64_t2 pos;
+    pfloat64_t height;
```

Comment:
add comment here that `height` can be `float32_t`.
I'll later handle in my gpu driven work, right now it wouldn't help with size of the struct due to padding, unless stored in a separate array (SoA style)